qchem/pennylane_qchem/qchem/obs.py

```diff
@@ -314,9 +316,13 @@ def observable(fermion_ops, init_term=0, mapping="jordan_wigner", wires=None):
 
     # Map the fermionic operator to a qubit operator
     if mapping.strip().lower() == "bravyi_kitaev":
-        return structure.convert_observable(bravyi_kitaev(mb_obs), wires=wires)
+        return structure.convert_observable(
```

Comment:
Oh.. is the qchem folder not being checked by black?

Reply:
Yes, this is weird. Perhaps using an outdated version of black?

Reply:
I don't think qchem and the qchem tests are being automatically checked with `black` - do we want to turn that on (I vote yes)?

Reply:
I vote yes as well, I think it was an oversight that it wasn't being checked!